### PR TITLE
onevpl-intel-gpu: Rename to vpl-gpu-rt

### PR DESCRIPTION
**Pull request by pjungkamp**
## Description of changes

Intel has merged the `oneapi-src` GitHub organization into the `intel` namespace. `oneVPL-intel-gpu` has been renamed to `vpl-gpu-rt` as part of this move. The repo at https://github.com/oneapi-src/oneVPL-intel-gpu now redirects to https://github.com/intel/vpl-gpu-rt.

I've updated the package `src` and `name` attributes and moved it from  `pkgs/by-name/on/onevpl-intel-gpu` to `pkgs/by-name/vp/vpl-gpu-rt`. I've also introduced a top-level alias with a warning for `onevpl-intel-gpu`.

There are no dependent packages. The wiki also does not document or refer to `onevpl-intel-gpu` (although it should probably).

Users of `onevpl-intel-gpu` will by notified by the warning on evaluation.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by PJungkamp** [comment]
Forgot to tag the maintainer: @evanrichter

**Comment by evanrichter** [comment]
I'd be glad to have you co-maintain, you've been submitting great patches! Tag me in that pr, I'll definitely approve

**Review by evanrichter — APPROVED** [review]

**Comment by PJungkamp** [comment]
Who is supposed to do a final review and merge this PR after maintainer approval. What is the procedure here?

**Comment by wegank** [comment]
Practically any committer who has noticed the PR. If not in a week, you can send the PR to https://matrix.to/#/#review-requests:nixos.org.

**Comment by huantianad** [comment]
I would love some good documentation on the intel gpu stack! Right now it's hard to know what packages I need to get hardware acceleration working in all my applications, ffmpeg, etc.

**Pull request merged**
